src/main/java/brandkon/ProductController.java

```diff
@@ -0,0 +1,41 @@
+package brandkon;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class ProductController {
+
+    private ProductService productService;
+
+    public ProductController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    //상품 목록 조회
+    @GetMapping("/products")
+    public List<ProductResponse> getAllProducts(@RequestParam(defaultValue = "0") long brandId){
```

Comment:
여기도 그냥 이렇게 두고 데이터가 안 들어오면 `null`이 되게 하는 것이 나을 것 같아요.
```suggestion
    public List<ProductResponse> getAllProducts(Long brandId){
```